setup/q2rtx.sh

```diff
@@ -2,21 +2,30 @@
 
 BIN_PREFIX="."
 
+if [[ -z "${XDG_DATA_HOME}" ]]; then
+	XDG_DATA_HOME="${HOME}/.local/share"
+fi
```

Comment:
Consider

```suggestion
XDG_DATA_HOME="${XDG_DATA_HOME:=${HOME}/.local.share}"
```

Personally I'd prefer also to  check if the directory actually exists as well, so also consider:
```suggestion
mkdir -p "${XDG_DATA_HOME:="${HOME}/.local/share"}"
```

Reply:
See stack overflow for explanation

https://stackoverflow.com/questions/2013547/assigning-default-values-to-shell-variables-with-a-single-command-in-bash

Reply:
It can be debated whether this is more concise or faster but either way, the directory needs to be made. 7ae9ce53.